crud/ProductManager/ProductManager/Classes/Manager.cs

```diff
@@ -0,0 +1,273 @@
+                result.UpdatedDate = (DateTime)reader["UpdatedDate"];
+                result.ProductId = (int)reader["ProductId"];
+                result.Status = (int)reader["Status"];
+                
+                connection.Close();
+            }
+            return result;        }
+
+        public void UpdateOrder(Order updatedOrder)
+        {
+            using (var connection = new NpgsqlConnection(_connectionString))
+            {
+                connection.Open();
+                var cmd = connection.CreateCommand();
+                cmd.CommandText = $"UPDATE public.\"Order\" " +
+                                  $"SET \"CreateDate\"='{updatedOrder.CreateDate}', " +
+                                  $"\"UpdatedDate\"='{updatedOrder.UpdatedDate}', " +
+                                  $"\"ProductId\"={updatedOrder.ProductId}, " +
+                                  $"\"Status\"={updatedOrder.Status}";
+                
+                cmd.ExecuteNonQuery();
+                connection.Close();
+            }
+        }
+
+        public void DeleteOrder(int orderId)
+        {
+            using (var connection = new NpgsqlConnection(_connectionString))
+            {
+                connection.Open();
+                
+                var cmd = connection.CreateCommand();
+                cmd.CommandText = $"DELETE FROM public.\"Order\" WHERE \"Id\"={orderId}";
+
+                cmd.ExecuteNonQuery();
+                connection.Close();
+            }         
+        }
+
+        public List<Product> GetAllProducts()
+        {
+            using (var connection = new NpgsqlConnection(_connectionString))
+            {
+                connection.Open();
+
+                var cmd = connection.CreateCommand();
+                cmd.CommandText = "SELECT * FROM public.\"Product\"";
+                
+                using (var reader = cmd.ExecuteReader())
+                {
+                    while (reader.Read())
+                    {
+                        var product = new Product
+                        {
+                            Id = (int)reader["Id"],
+                            Name = (string)reader["Name"],
+                            Description = (string)reader["Description"],
+                            Weight = (double)reader["Weight"],
+                            Width = (double)reader["Width"],
+                            Height = (double)reader["Height"],
+                            Length = (double)reader["Length"]
+                        };
+
+                        _products.Add(product);
+                    }
+                }
+                
+                connection.Close();
+                return _products;
+            }
+        }
+
+        public List<Order> GetOrdersByFilter(int filterYear, int filterMonth, string filterStatus, int filterProductId)
+        {
+            var filteredOrders = _orders;
```

Comment:
remove '_orders' collection from code, we don't need it at all, create new collection here instead